README.md

```diff
@@ -259,6 +259,46 @@ Additional configuration are created in /etc/nginx/conf.d/
                - root "/usr/share/nginx/html"
                - index index.html
 ```
+8) Site configuration using a custom template.
+Instead of defining a site config file using a list of attributes,
+you may use a hash that indicates an alternate template.
+Additional hash elements are accessible inside the template through
+the `nginx_sites[item]` hash.
+```yaml
+- hosts: all
+
+  roles:
+  - role: nginx
+    nginx_sites:
+      bar:
+        template: bar.conf.j2
+        server_name: bar.example.com
```

Comment:
I just understand the trick with your further explication.
I will merge it in fact, can be little confusing but it's very clever.